### PR TITLE
Mark all compiled JS as binary

### PR DESCRIPTION
**Pull request by nickvergessen**
Signed-off-by: Joas Schilling <coding@schilljs.com>

### CODE REVIEW

**Review by rullzer — APPROVED** [review]

File: .gitattributes [hunk 1]
```diff
@@ -1,8 +1,8 @@
 /core/js/dist/*.js binary
 /core/js/dist/*.js.map binary
 
-/apps/accessibility/js/accessibility.js binary
-/apps/accessibility/js/accessibility.js.map binary
+/apps/accessibility/js/*.js binary
+/apps/accessibility/js/*.js.map binary
```
Comment (skjnldsv) [inline .gitattributes:5]: We could probably even do :stuck_out_tongue: 
```suggestion
/apps/*/js/*.js binary
/apps/*/js/*.js.map binary
```

Comment (nickvergessen) [reply .gitattributes:5]: But we still have non-compiled js files :fearful:

Comment (skjnldsv) [reply .gitattributes:5]: Ah right :cry:

**Review by skjnldsv — APPROVED** [review]

**Pull request merged**